ta_services/keymaster/include/keymaster_ta.h

```diff
@@ -17,4 +17,20 @@
  */
 #define KEYMASTER_CMD_ADD_RNG_ENTROPY	0
 
+/*
+ * Configure keymaster with KM_TAG_OS_VERSION and
+ * KM_TAG_OS_PATCHLEVEL. Until keymaster is configured, all other
+ * functions return TEE_ERROR_NOT_CONFIGURED. Values are only accepted
+ * once. Subsequent calls return TEE_SUCCESS, but do nothing.
+ *
+ * in	params[0].value.a: KM_TAG_OS_VERSION
+ * in	parmas[0].value.b: KM_TAG_OS_PATCHLEVEL
```

Comment:
I think we need to decide how to return suitable error codes. Best control over that is to return `enum ErrorCode` as an out value parameter. I'm planning to do that for `GenerateKey()` and `AttestKey()`.

Reply:
If you don't mind changing this later keep it as it is for now.

Reply:
Ok. Will change later.